### PR TITLE
Add Podcast component

### PR DESCRIPTION
**Pull request by joeytepp**
## Proposed Change

Closes: #113 

Add the `Podcast` component to the qhacks website!

### How did you do this?
Created a new `Podcast` component, added it to the index page and updated tests

### Why are you choosing this approach?
Keep things modular, keep the momentum going

### Any open questions you want to ask code reviewers?
No

### List of changes:

  - Add new `Podcast` component
  - Add component to `index` page
  - Update tests

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new tests for my core changes (where applicable).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit ab0a6d3a4ad9a4fe1510d41ef95effcf14802e99

https://deploy-preview-122--qhacks-website.netlify.com

File: src/components/Podcast.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        gridTemplateColumns: "min-content auto",
+        "@media (max-width: 1128px)": { display: "block" },
+        padding: "40px 28px 40px 28px",
+        width: "100%"
+      }}
+    >
+      <div css={{ "@media (max-width: 1128px)": { textAlign: "center" } }}>
+        <img
+          src={podcast}
+          css={{
```
Comment (rosslh) [inline src/components/Podcast.js]: Headphones look a bit too close to the left side of the container compared to the mocks

File: src/components/Podcast.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        css={{
+          textAlign: "left",
+          paddingLeft: "64px",
+          "@media (max-width: 1128px)": {
+            paddingLeft: "0px",
+            paddingTop: "46px"
+          }
+        }}
+      >
+        <h1 css={{ paddingBottom: "24px" }}>Listen to our podcast!</h1>
```
Comment (rosslh) [inline src/components/Podcast.js]: There should be more whitespace above heading to match mocks

File: src/components/Podcast.js (hunk 1 above)
Comment (joeytepp) [reply src/components/Podcast.js]: Nice catch 👀

File: src/pages/index.js [hunk 3]
```diff
@@ -52,5 +53,6 @@ export default () => (
     <Speakers />
     <EventSchedule />
     <FAQs />
+    <Podcast />
```
Comment (RobertWSaunders) [inline src/pages/index.js]: Every time a new component gets added to this I get so happy! 👍 😃

File: src/components/MailingListSignup.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import circuits from "../assets/img/circuits.png";
+import circuits from "../assets/img/circuitsRed.png";
```
Comment (RobertWSaunders) [inline src/components/MailingListSignup.js]: Nice change!

**Comment by RobertWSaunders** [comment]
I'll merge and investigate why Coveralls is not reporting!

**Pull request merged**